crypto/cipher/aes_wrap.c

```diff
@@ -219,9 +219,11 @@ static srtp_err_status_t srtp_aes_wrap_set_iv (srtp_aes_wrap_ctx_t *c, const uin
     /*
      * Allocate space for the IV
      */
-    c->alternate_iv = malloc(c->alternate_iv_len);
     if (!c->alternate_iv) {
```

Comment:
This looks like a valid concern.  However, rather than performing this check here, should we simply allocate the space in srtp_aes_wrap_alloc() and free it in srtp_aes_wrap_dealloc()?